.gitignore

```diff
@@ -18,3 +18,6 @@ out
 # accidental upstream patch release breakage before users do.
 examples/*/Cargo.lock
 test/src/sdk-harness/Cargo.lock
+
+# These are unneeded mac generated files
+.DS_Store
```

Comment:
I think it's generally better practice to have platform/editor/user-specific files under your own local gitignore file, otherwise we open the floodgates to filling the `.gitignore` with files unrelated to the project. [This SO post discusses how you can setup a local gitignore](https://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file). All that said, as `.DS_Store` is such a common culprit I'm not against leaving it in.

Reply:
I'll check back in tonight, lmk if I should remove the `.DS_Store`

Reply:
Yeah, take it out.